music21/romanText/writeRoman.py

```diff
@@ -332,11 +360,14 @@ def rnString(measureNumber: int,
     '''
 
     if inString:
```

Comment:
In definition of rnString, change type to t.Union[int, str]

Reply:
But it looks like you're concating int(measure.number) + measure.suffix only to separate them later.  why not pass in the suffix separately?

I didn't look carefully enough at `rnString()` when this went in.  It's quite inefficient.  It seems to me that it really should be an object whose __str__ gives the correct result so you don't need to keep passing in a the same string to concat to and read from over and over.  But that's a later/larger refactor.

Reply:
Thanks all.
+1 for sorting out `rnString`.
Mea culpa for how it looks as it stands though I suspect it ended up that way as one of those solutions that seeks to please everyone ... but ends up half way between different solutions and ends up pleasing no one ;)
See #1393 for an alternative that removes it completely.
I'm happy to advance this sometime.